nmkapp/templates/request_display_name.html

```diff
@@ -7,29 +7,29 @@
 
 <h1 style="text-align: center;">{% trans "SHARKZ.BET - ADD PASSWORD" %}</h1>
 {% if nonvalid %}
-{% trans "Invalid request to add password" %}
+{% trans "Invalid request to select display name" %}
 {% else %}
     {% if reset %}
-    {% trans "Password added successfully" %}
+    {% trans "Display name set successfully" %}
     <br/>
     <br/>
     <div>
         <a href="/login">{% trans "Back to login" %}</a>
     </div>
     {% else %}
-    <form method="post" action="/requestpassword">
```

Comment:
Jel treba da imamo ovde neki tekst koji objašnjava poentu i zašto ovo tražimo? Jedna rečenica ohrabrujuća, tipa: "Grest, you are now logged. You just need to pick a name you will be knowb around here"... Lupam